### PR TITLE
Gemini 1.5 multimodal snippets

### PR DESCRIPTION
**Pull request by glaforge**
Fixes b/330496709, b/330500930, b/330497454, b/330500054, adding multimodal snippets using Gemini 1.5

### CODE REVIEW

File: vertexai/snippets/src/main/java/vertexai/gemini/MultimodalAllInput.java
Comment (Sita04) [inline vertexai/snippets/src/main/java/vertexai/gemini/MultimodalAllInput.java:40]: Should this description be updated?

File: vertexai/snippets/src/main/java/vertexai/gemini/MultimodalVideoAudioInput.java
Comment (Sita04) [inline vertexai/snippets/src/main/java/vertexai/gemini/MultimodalVideoAudioInput.java:19]: Kindly update this region tag, as its also used in `MultimodalAllInput`

File: vertexai/snippets/src/main/java/vertexai/gemini/MultimodalVideoAudioInput.java
Comment (Sita04) [inline vertexai/snippets/src/main/java/vertexai/gemini/MultimodalVideoAudioInput.java:40]: Comment should be updated to reflect the method.

File: vertexai/snippets/src/test/java/vertexai/gemini/SnippetsIT.java
Comment (Sita04) [inline vertexai/snippets/src/test/java/vertexai/gemini/SnippetsIT.java:2]: The copyright year refers to when the file was initially created. Hence, please revert the year.

**Review by Sita04 — CHANGES_REQUESTED** [review]
Thanks for the snippets!

File: vertexai/snippets/src/main/java/vertexai/gemini/MultimodalAllInput.java
Comment (glaforge) [reply vertexai/snippets/src/main/java/vertexai/gemini/MultimodalAllInput.java:40]: I'll put `A request containing a text prompt, a video, and a picture.`

File: vertexai/snippets/src/test/java/vertexai/gemini/SnippetsIT.java
Comment (glaforge) [reply vertexai/snippets/src/test/java/vertexai/gemini/SnippetsIT.java:2]: Good to know, I'll revert.

File: vertexai/snippets/src/main/java/vertexai/gemini/MultimodalVideoAudioInput.java
Comment (glaforge) [reply vertexai/snippets/src/main/java/vertexai/gemini/MultimodalVideoAudioInput.java:19]: Well spotted. Copy and paste error. It's `generativeaionvertexai_gemini_all_modalities` I should have used in `MultimodalAllInput`.

File: vertexai/snippets/src/main/java/vertexai/gemini/MultimodalVideoAudioInput.java
Comment (glaforge) [reply vertexai/snippets/src/main/java/vertexai/gemini/MultimodalVideoAudioInput.java:40]: I'll put `Analyzes the given video input, including its audio track.` which is more explicit.

**Review by gericdong — APPROVED** [review]

**Review by Sita04 — APPROVED** [review]

**Comment by Sita04** [comment]
@glaforge Requesting to wait until this PR has been merged:https://github.com/GoogleCloudPlatform/java-docs-samples/pull/9303

**Pull request merged**
